src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -11,9 +11,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="$(MlDotNetPackageVersion)" />
-    <PackageReference Include="Microsoft.ML.LightGBM" Version="$(MlDotNetPackageVersion)" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="$(MlDotNetPackageVersion)" />
+    <PackageReference Include="Microsoft.ML" Version="1.4.0-preview3-28218-2" />
+    <PackageReference Include="Microsoft.ML.LightGBM" Version="1.4.0-preview3-28218-2" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.4.0-preview3-28218-2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.Recommender\Microsoft.ML.Recommender.csproj" />
```

Comment:
Why aren't you using a `PackageReference` to Recommender here, same as the other 3?